config/kind-ci/noReservedResources.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  template:
+    spec:
+      containers:
+        - name: manager
+          resources:
+            requests:
+              cpu: 0m
```

Comment:
does this disable the requests (OK) or sets them to 0? I'm worried about side effects/unwanted behaviour when we DO set requests with zero value, which I'm not sure how common it is

Reply:
It is the simplest way for patching.
I need to check how to delete the ```resources``` stanza with patching.

Reply:
I don't mind using this way if the end result is the same as removing the requests. If we can demonstrate this referring to the docs/APIs then it's fine and no need to change.

Reply:
I'm always in favor of doing things the right way. 
And I learned a new thing on the way. check the commit, it's kinda cool.